lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js

```diff
@@ -121,3 +132,36 @@ test('When all previous required questions have been answered - and a url is ret
   t.equals(expectedInstance.redirect, undefined, 'it should not return an unanswered page to redirect to')
   t.end()
 })
+
+test('it attaches a user submission', async t => {
+  const pageSummaryController = proxyquire('./page.summary.controller', {
+    '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
+    '../../../../client/submitter/submitter': {submitterClient: submitterClientSpy},
+    '../../../../constants/constants': {ATTACH_USER_SUBMISSION: true}
+  })
+
+  await pageSummaryController.postValidation({}, userData)
+
+  const submissions = submitterClientSpy.getCall(0).args[0].submissions
+  t.equals(submissions[0]['recipientType'], 'user')
+  t.equals(submissions[0].attachments.length, 1)
+  submitterClientSpy.resetHistory()
+  t.end()
+})
+
+test('it does not attach a user submission', async t => {
+  const pageSummaryController = proxyquire('./page.summary.controller', {
+    '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
+    '../../../../client/submitter/submitter': {submitterClient: submitterClientSpy},
+    '../../../../constants/constants': {ATTACH_USER_SUBMISSION: false}
+  })
```

Comment:
Unsure how to remove this duplication.  Brief research suggests this isn't possible without setting global state which can be fragile